### PR TITLE
no arch docs

### PR DESCRIPTION
**Pull request by ocefpaf**
@jjhelmus and @isuruf can you guys take a quick look at this text?

### CODE REVIEW

File: src/meta.rst [hunk 1]
```diff
@@ -99,6 +99,26 @@ or if the package uses ``cimport``.
...
+only once on ``CircleCI``.
+
+To use that just add ``noarch: python`` in the build section like,
+
+.. code-block:: yaml
+
+    build:
+      noarch: python
+
+and re-render with the feedstock with ``conda-smithy`` >=2.4.0
```
Comment (isuruf) [inline src/meta.rst]: and `conda-build-all >=1.1.0`

Comment (ocefpaf) [reply src/meta.rst]: I don't want to confuse the end user with that information here. The use of `conda-build-all` is abstracted away in the feedstock and in `conda-smithy` dependency chain, right? So no need to mention and get users going into a wild goose chase.

Comment (isuruf) [reply src/meta.rst]: I guess we should update `conda-smithy`'s `meta.yaml` then

Comment (ocefpaf) [reply src/meta.rst]: Definitely!

Comment (isuruf) [reply src/meta.rst]: https://github.com/conda-forge/conda-smithy-feedstock/pull/81

Comment (ocefpaf) [reply src/meta.rst]: You are too fast :smile:

**Comment by jjhelmus** [comment]
Sorry for the late response, the text looks great to me.  Thanks for writing this up @ocefpaf.

**Comment by jakirkham** [comment]
Also late. Probably worth noting that it can't have any OS specific dependencies (see below).

```yaml
    - menuinst  # [win]
```

**Pull request merged**